### DISCUSSION TITLE
Cannot sign in to vercel

### DISCUSSION THREAD

**Discussion by Queen-esther01** [vercel/community#1044] [Help] [answered]
For some reason I am unable to sign in to vercel. I signed into vercel sometime this month with a friend's account to help him set up and now I can't log back in to my own account. I have revoked access on Github, deleted Github app and cleared cache. Now I am getting an error that says there is no  Vercel account linked with my GitHub account, all apps I have on vercel are still online, any idea on how to solve this would be appreciated, thank you.

**Comment by amyegan** [comment, marked as answer]
Hi @Queen-esther01! If you are not able to log in with your email address, please send a message to [registration@vercel.com](mailto:registration@vercel.com). Be sure to include the GitHub username in your message.

**Reply by Queen-esther01** [reply]
Thank you @amyegan Sending an email right away.
